src/Browser/Json.php

```diff
@@ -83,15 +86,16 @@ public function assertHas(string $selector): self
      */
     public function assertMissing(string $selector): self
     {
-        try {
-            $this->search("length({$selector})");
-        } catch (\RuntimeException $e) {
-            Assert::pass();
+        $prefix = 1 === preg_match('#^\[#', $selector) ? '' : '@.';
+        $exploded = explode('.', $prefix.$selector);
+        $key = array_pop($exploded);
+        $keySelector = implode('.', $exploded);
 
-            return $this;
-        }
+        dump($selector, $keySelector, $key);
```

Comment:
don't forget to remove.

Reply:
Oups, that's why I waited for feedback on the issue :slightly_smiling_face: